modules/plottinggl/src/plotters/scatterplotgl.cpp

```diff
@@ -64,24 +66,24 @@ ScatterPlotGL::Properties::Properties(std::string_view identifier, std::string_v
     , minRadius_("minRadius", "Min Radius", 0.1f, 0, 10, 0.01f)
     , tf_("transferFunction", "Transfer Function",
           TransferFunction({{0.0, vec4(1.0f)}, {1.0, vec4(1.0f)}}))
-    , color_("color", "Color", vec4(1, 0, 0, 1), vec4(0), vec4(1), vec4(0.1f),
-             InvalidationLevel::InvalidOutput, PropertySemantics::Color)
-    , hoverColor_("hoverColor", "Hover Color", vec4(1.0f, 0.906f, 0.612f, 1))
-    , selectionColor_("selectionColor", "Selection Color", vec4(1.0f, 0.769f, 0.247f, 1))
+    , color_("defaultColor", "Color", util::ordinalColor(1.0f, 0.0f, 0.0f, 1.0f))
```

Comment:
I know that nothing really has changed, but IMHO red is not an optimal default color for selection.

Reply:
It is not the selection color, it refers to the default color for data points.
See https://github.com/inviwo/inviwo/blob/1ca7e90e221208f40af249687e8b5c33d2fc06c4/modules/plottinggl/src/plotters/scatterplotgl.cpp#L67-L68